supplementary_style_guide/style_guidelines/mananged-services.adoc

```diff
@@ -0,0 +1,107 @@
+
+Ensure product documentation meets Section 508 compliance for accessibility. Accessibility standards are derived from the link:https://www.w3.org/TR/WCAG20/[Web Content Accessibility Guidelines (WCAGA)] and the link:https://www.access-board.gov/ict/[ICT Accessibility 508 Standards and 255 Guidelines].
+
+=== Images
+
+*Accessibility criteria:* The document provides text equivalents for every diagram, image, or other non-text element. Avoid using images of text instead of actual text.
+
+Evaluation:
+
+* All icons, images, diagrams, and non-text elements that convey information must have meaningful alternative-text descriptions.
+** For icons, use alternative text that describes what the icon does rather than what it looks like. For example, the alt text for a + icon is Add rather than Plus Sign.
+* The document is free from images of text; for example, a screen capture image of an informational table. Make sure that actual text is used to convey information, rather than images of text.
+
+=== Links and hypertext
+
+*Accessibility criteria:* The purpose of each link can be determined from the link text alone, or from the link text together with the link context. Links lead to the correct target locations within the document, or to a valid external Web page location (no broken links).
+
+* All link URLs and cross-reference links must provide descriptive text that conveys information about the content of the target linked page prior to clicking the link. Do not use a generic phrase like “click here.”
+
+Examples:
+
+* A link to a different section in the same document shows the section title.
+* An external link provides the site name, or title of the target web page.
+
+* URLs must link to correct and valid Web destinations.
+
+=== Tables
+
+*Accessibility criteria:* When a document includes tables, accessibility tools such as a screen reader can programmatically determine the location of the content in each row and column. Screen readers interact with a data table based on the number of columns and rows, and provide table navigation to the user by reading row and column headers in the order they occur.
+
+* All tables must have a simple, logical reading order from left to right, top to bottom.
+* Avoid tables with irregular headers. Tables with multi-level headers, or spanned rows and cells, don’t provide a clear horizontal or vertical association between header and data cells, and are too complex for accessibility tools to interpret.
+** See the example in link:https://www.w3.org/WAI/tutorials/tables/irregular/[Tables with irregular headers]
+* Avoid blank header cells. Older screen readers often do not handle blank header cells correctly. Simplify tables to remove empty header cells, or add row and column text to each header cell.
+* Use TH tag in all header cells, and TD tag in all data cells.
+* Label tables with a caption. See link:https://docs.asciidoctor.org/asciidoc/latest/tables/add-title/[Add a Title to a Table] for more information.
```

Comment:
Suggest specifying that this link is about adding titles using AsciiDoc. Maybe "For more information adding titles using AsciiDoc, see ..."

Reply:
I agree with Andrea. And I think this guidance is correct, but I think it might help to clarify that a title is a caption. It says label tables with a caption and then sends you to how to add a title to a table. Maybe we could say add a title/caption. 

Bob wrote a blog post about this a while ago and included a bit about how to add a summary also. https://source.redhat.com/groups/public/ccs/ccs_blog/accessibility_of_tables

Reply:
Updated